lua/docs/content/reference/index.yml

```diff
@@ -60,6 +60,7 @@ blocks:
     - list:
         - "<a href=\"/reference/array\">array</a>"
         - "<a href=\"/reference/boolean\">boolean</a>"
+        - "<a href=\"/reference/condition\">condition</a>"
```

Comment:
`condition` is not a Lua data type 🤔 ,
but this PR is a nice addition.

would you accept to add it in a new list below ?
like so :

```yaml
    - text: "Lua concepts:"
     - list:
         - "<a href=\"/reference/condition\">condition</a>"
```

Reply:
Yes no problem! it's my bad, I had misunderstood the type of category that it was.
We could add the loop reference in this new part

Reply:
Thanks.
Indeed. 🙂